components/MetricsTable.tsx

```diff
@@ -0,0 +1,109 @@
+import debugFactory from 'debug'
+import { toInt } from 'qc-to_int'
+import React, { MouseEvent, useState } from 'react'
+
+import Menu from 'semantic-ui-react/dist/commonjs/collections/Menu'
+import Table from 'semantic-ui-react/dist/commonjs/collections/Table'
+
+import { MetricBare } from '@/models/index'
+
+const debug = debugFactory('abacus:components/MetricsTable.tsx')
+
+const PER_PAGE_DEFAULT = 25
+
+interface Props {
+  metrics: MetricBare[]
+  // The zero-based page of data to display. Defaults to 0.
+  initialPage?: number
+  // The number of rows/items to display per page. Defaults to PER_PAGE_DEFAULT.
+  perPage?: number
+}
+
+/**
+ * Renders a table of "bare" experiment information.
+ */
+const MetricsTable = (props: Props) => {
+  debug('MetricsTable#render')
+  const { metrics, initialPage = 0, perPage = PER_PAGE_DEFAULT } = props
+  const [page, setPage] = useState(initialPage)
+
+  const pageCount = Math.ceil(metrics.length / perPage)
+  const pageNums = Array.from(Array(pageCount).keys()).map((num) => num + 1)
+
+  function handlePageStep(event: MouseEvent<HTMLAnchorElement>) {
+    debug('MetricsTable#handlePageStep')
+    event.preventDefault()
+    event.stopPropagation()
+    setPage(page + toInt(event.currentTarget.dataset.step))
+  }
```

Comment:
This could probably be a higher order function so you don't have to rely on the rendered state:

```suggestion
  function handlePageStep(nextPage) => (event: MouseEvent<HTMLAnchorElement>) => {
    debug('MetricsTable#handlePageStep')
    event.preventDefault()
    event.stopPropagation()
    setPage(nextPage)
  }
```